tx_btree.go

```diff
@@ -62,6 +62,10 @@ func (tx *Tx) PutIfNotExists(bucket string, key, value []byte, ttl uint32) error
 
 // PutIfExits set the value for a key in the bucket only if the key already exits.
 func (tx *Tx) PutIfExists(bucket string, key, value []byte, ttl uint32) error {
+	return tx.updateIfExists(bucket, key, value, ttl, false)
+}
+
+func (tx *Tx) updateIfExists(bucket string, key, value []byte, ttl uint32, updateTTLOnly bool) error {
```

Comment:
We already have helper method named `update` and so on, maybe we can edit and reuse them?